app/views/groups/show.html.haml

```diff
@@ -1,3 +1,78 @@
-#content.section--primary{ tabindex: "0" }
-  .container
-    %h1= "#{title(@group.name)} (#{@group.date_range})"  if @group.name.present?
+- page_title = title("#{@group.name}")
+
+- if @group.name.present?
+
+  #content.section--primary{ tabindex: '0' }
+    .container
+      %h1= page_title
+
+      %p.lead
+        - if @group.formal_body? && @group.select_committee?
+          - if @group.joint?
+            = "#{t('joint_committee')} #{@group.committee_type} #{t('committee')}"
+          - else
+            - if @group.houses.size == 1 && @group.houses.first.name == "House of Commmons"
+              = "#{t('commons')} #{@group.committee_type} #{t('committee')}"
+            - else
+              = "#{t('lords')} #{@group.committee_type} #{t('committee')}"
+        - else
+          = @group.name
+
+  - if @group.formal_body? && @group.remit.present?
+    %section
+      .container
+        %h2= "#{t('.about_the').capitalize} #{t('committee')}"
+        %p= "#{@group.remit}"
+
+  - if @group.member_count
+    %section
+      .container
+        %h2= "#{t('committee').capitalize} #{t('member_plural')}"
+
+        - if @chair_people.any?
+          - @chair_people.each do |chair_person|
+            %ol.list--block
+              %li
+                %card
```

Comment:
This should be `.card` instead of `%card`; `%card` would mean you are calling the HTML *card* element, which doesn't exist. Other than that, this part looks corrected good. I haven't had a look at the rest, which I'll leave to a backend developer. Thanks for implementing this.